tensorflow/python/training/monitored_session.py

```diff
@@ -306,9 +307,10 @@ def MonitoredTrainingSession(master='',  # pylint: disable=invalid-name
     hooks: Optional list of `SessionRunHook` objects.
     chief_only_hooks: list of `SessionRunHook` objects. Activate these hooks if
       `is_chief==True`, ignore otherwise.
-    save_checkpoint_secs: The frequency, in seconds, that a checkpoint is saved
-      using a default checkpoint saver. If `save_checkpoint_secs` is set to
-      `None`, then the default checkpoint saver isn't used.
+    save_checkpoint_secs: The frequency, in number of global steps, that a
```

Comment:
Please fix comment. Please specify the behavior when both are set.

Reply:
fixed!